src/main/java/com/prajina/academy/transformer/CourseTransformer.java

```diff
@@ -0,0 +1,38 @@
+package com.prajina.academy.transformer;
+
+import org.springframework.stereotype.Component;
+
+import com.prajina.academy.model.Course;
+
+@Component
+public class CourseTransformer {
```

Comment:
If you don't put a convert method in the Course Object itself.  you can put conversions in a Manager class for that object.